AIO/src/org/aio/activities/quests/RomeoAndJuliet.java

```diff
@@ -150,16 +147,27 @@ private void talkToJuliet() throws InterruptedException {
         }
     }
 
-    private void getItemFromObject(Area place, String itemName, String objectName, String interaction) throws InterruptedException {
-        if (place.contains(myPlayer())) {
-            RS2Object object = getObjects().closest(o -> o.getName().equals(objectName) && o.hasAction(interaction));
+    private void getBerry(){
+        String interaction = "Pick-from";
+        if(BERRIES.contains(myPlayer())){
+            List<RS2Object> objects = getObjects().filter(o -> o.getName().equals("Cadava bush") && o.hasAction(interaction));
+            //Antipattern pick a random Bush
```

Comment:
I personally don't use anti-pattern/anti-ban, but might be good, just keep in mind we should have more generic functions and pull them out into a helper class.

Reply:
True that, antiban is fake anyway lol